test/suites/remote.sh

```diff
@@ -147,8 +148,9 @@ test_remote_url_with_token() {
 }
 
 test_remote_admin() {
-  ! lxc_remote remote add badpass "${LXD_ADDR}" --token badtoken || false
-  ! lxc_remote list badpass: || false
+  lxc_remote remote add badtoken "${LXD_ADDR}" --token badtoken 2>&1 | grep -F "Error: Failed to decode trust token:"
+  ! lxc_remote remote add badtoken "${LXD_ADDR}" --token badtoken || false
```

Comment:
Is the first test already sufficient? The second one seems the same, just not checking the error message?